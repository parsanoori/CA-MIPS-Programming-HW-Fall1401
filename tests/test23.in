2
929927613
643919475