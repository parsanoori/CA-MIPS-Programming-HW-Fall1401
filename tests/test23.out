643919475
929927613